cukas.js

```diff
@@ -106,6 +107,22 @@ class CukasGame {
             this.players[defencePlayerId].hand.push(this.deck.getRandomSet(Math.abs(this.players[attackPlayerId].hand.length - 6))); //make sure defence has 6 cards
         }
         //console.log(this.players);
+
+    }
+    compareCards(A,B){//1 - a>b -1 - a<b 0 - a=b
+      if(A.type==Card.TYPE_JOKER)return 1;
```

Comment:
consider adding typechecks using `variable instanceof Class`

Reply:
this isn't a class though, or at least I think it isn't